CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -117,19 +117,6 @@ public Task<bool> IsEmailTakenAsync(string email)
             return _unitOfWork.AccountRepository.IsEmailTakenAsync(email);
         }
 
-        public async Task<AccountDto> UpdateAccountCredentialsAsync(Account account)
-        {
-            
-            account.Salt = GenerateSalt();
-            account.Password = HashPassword(account.Password, account.Salt);
-
-            _unitOfWork.AccountRepository.UpdateAccountCredentials(account);
-
-            await _unitOfWork.CommitAsync();
-
-            return _mapper.Map<AccountDto>(account);
-        }
-
```

Comment:
why did u deleted rhis method?

Reply:
> why did u deleted rhis method?

This method is not usable anymore. We don't need it.